sklearn/tree/_export.py

```diff
@@ -943,6 +944,14 @@ def export_text(
         A list of length n_features containing the feature names.
         If None generic names will be used ("feature_0", "feature_1", ...).
 
+    class_names : list or None, default=None
+        Names of each of the target classes in ascending numerical order.
+        Only relevant for classification and not supported for multi-output.
+
+        - if `None`, the class names are delegated to `decision_tree.classes_`;
+        - if a list, the number of items should be the same as in
+          `decition_tree.classes_` and will be used.
+
```

Comment:
This needs a `.. versionadded:: 1.3` directive to indicate that the parameter was added in 1.3.